pkg/maintainer/bitcoin_difficulty.go

```diff
@@ -0,0 +1,347 @@
+package maintainer
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+)
+
+const (
+	// Default value for back-off time which should be applied when the Bitcoin
+	// difficulty maintainer is restarted. It helps to avoid being flooded with
+	// error logs in case of a permanent error in the Bitcoin difficulty
+	// maintainer.
+	defaultRestartBackoffTime = 120 * time.Second
+
+	// Default value for back-off time which should be applied when there are no
+	// more Bitcoin epochs to be proven because the difficulty maintainer is
+	// up-to-date with the Bitcoin blockchain or there are not enough blocks yet
+	// to prove the new epoch.
+	defaultIdleBackOffTime = 60 * time.Second
```

Comment:
```suggestion
	bitcoinDifficultyDefaultIdleBackOffTime = 60 * time.Second
```

Reply:
Done in f360163494a01b05f91c07f8ba4cafafb1e9951d.